Solutions/Marain.ContentManagement.Api.Services/Marain/Cms/Api/Services/Internal/ContentMapper.cs

```diff
@@ -0,0 +1,51 @@
+﻿// <copyright file="ContentMapper.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.Cms.Api.Services.Internal
+{
+    using Menes;
+    using Menes.Hal;
+    using Menes.Links;
+
+    /// <summary>
+    /// Maps <see cref="Content"/> to and from a <see cref="HalDocument"/>.
+    /// </summary>
+    public class ContentMapper : IHalDocumentMapper<Content, IOpenApiContext>
```

Comment:
I've renamed the mapper classes to contain the name of the thing they map to, as defined in the YAML - I think this makes most sense. It happens that all of those responses are HAL documents by definition, but I think it makes less sense now to include this in the names.

I haven't included the name of the "from" type in the mapper names as I think this would lead to unpleasantly long names, e.g. `ContentSummariesWithStateResponseMapper` would become `ContentSummariesWithStateToContentSummariesWithStateResponseMapper`. I think because we have pretty decent alignment of internal names to response types at the moment it's unnecessary. Thoughts?